app/src/main/java/emu/skyline/AppDialog.kt

```diff
@@ -101,6 +154,66 @@ class AppDialog : BottomSheetDialogFragment() {
             shortcutManager.requestPinShortcut(info.build(), null)
         }
 
+        val saveFolderPath = savesFolderRoot + item.titleId
+        val saveExists = File(saveFolderPath).exists()
+
+        binding.deleteSave.isEnabled = saveExists
+        binding.deleteSave.setOnClickListener {
+            AlertDialog.Builder(requireContext())
+                .setTitle(getString(R.string.delete_save_confirmation_message))
+                .setMessage(getString(R.string.action_irreversible))
+                .setPositiveButton(getString(R.string.yes)) { dialogInterface, _ ->
+                    File(saveFolderPath).deleteRecursively()
+                    binding.deleteSave.isEnabled = false
+                    binding.exportSave.isEnabled = false
+                    dialogInterface.dismiss()
+                }.setNegativeButton(getString(R.string.no)) { dialogInterface, _ ->
+                    dialogInterface.cancel()
+                }.show()
+        }
+
+        binding.importSave.setOnClickListener {
+            documentPicker.launch(arrayOf("application/zip"))
+        }
+
+        binding.exportSave.isEnabled = saveExists
+        binding.exportSave.setOnClickListener {
```

Comment:
Definitely encapsulate this logic to a separate `exportSave` function, the `onViewCreated` method is already pretty long, no need to make it bigger.